### DISCUSSION TITLE
Prime Button?

### DISCUSSION THREAD

**Discussion by jtaylor54** [nebhead/PiFire#112] [Ideas]
This is so cool. I have a Pitboss Copperhead vertical smoker and the controller fried. I would really like to replace it with this to have much more control over my smoke sessions. One thing my old controller had was a prime button to prime the empty auger feed tube. Does that exist with the PiFire?

Also if anyone has an extra V3 board they would like to sell EasyEDA keeps telling me invalid PCB so I can't order one.

**Comment by nebhead** [comment]
Glad you found us!  If you're just filling the auger the first time running the grill, you can go to manual mode in the admin tab and run the auger manually until you have pellets popping out.  I'm assuming this is not something that you'd need to do more frequently, right? 

Tagging @weberbox to raise awareness on the EasyEDA issue with the v3 board.

**Reply by weberbox** [reply]
A prime button was also on my table top PitBoss. I never used it but I thought it was a cool idea. I contemplated adding it to PiFire but have not really looked at it yet. It could be a cool feature I guess. 

I will take a look at the PCB when I get a chance. Not sure why there is an error I have not changed anything as far as I remember. 

If I still have any extra PCB's I can send one to you. I will take a look this week.

**Reply by jtaylor54** [reply]
@nebhead I do use it infrequently so the manual method should work just fine. 

@weberbox Thanks for the offer of the extra V3 board.

**Reply by weberbox** [reply]
@jtaylor54 I checked the v3 PCB and it seems to be working fine when I attempt to order it so I am not sure why you are getting an error. 

I do have an extra PCB but I forgot to ask if you are in the USA or not.

**Reply by NessK** [reply]
> I do have an extra PCB but I forgot to ask if you are in the USA or not.

@jtaylor54 If you are not in the US, I have a spare in EU.

**Reply by tukutt** [reply]
> > I do have an extra PCB but I forgot to ask if you are in the USA or not.
> 
> @jtaylor54 If you are not in the US, I have a spare in EU.

I @jtaylor54 , I'm in Europe ! Have you always spare ?

**Reply by NessK** [reply]
@tukutt -  I have spare PCB in the EU that I will gladly send you if you pay shipping and promise to post a picture once your build is done! :)
I have emailed you

**Comment by jtaylor54** [comment]
Sorry I'm away now but the next day I was able to order so thanks for the offers but I'll be okay and have some extra myself

Sent from Yahoo Mail on Android 
 
  On Sun, Aug 21, 2022 at 1:36 AM, ***@***.***> wrote:   



I do have an extra PCB but I forgot to ask if you are in the USA or not.


@jtaylor54 If you are not in the US, I have a spare in EU.

**Comment by nebhead** [comment]
FYI - I think I'm going to tackle adding a Priming feature in the coming days.  I see there is some value in having a few pellets in the fire-pot when starting a cook.  It tends to kick off a lot of really nice smoke right in the beginning - with the only drawback being that it also tends to spike the temperature.  With that said, I think it's a really good idea and wanted to add it in for anyone who might want to use it (including myself).  

Anyway, just thought I would throw this out there.  Look for this coming soon, in the development branch!

**Reply by jtaylor54** [reply]
I am interested. Are you thinking another button or using the existing buttons?

**Reply by nebhead** [reply]
Added a button to the Web UI and have added an option to the display menu, using existing buttons.  I fighting a bug in the display menu at the moment, but the prime feature itself I have running great.  Hopefully will have this pushed to the development branch for others to try out later today.  

WebUI Button:
![image](https://user-images.githubusercontent.com/19876886/194090334-8d273b01-456e-46c7-ba7e-e50957820ee5.png)
_The drop-up button has options for priming with different quantities, or priming and starting._

Display Menu Item:
![image](https://user-images.githubusercontent.com/19876886/194090156-927b45d2-de0a-4ac5-ad34-12bca7996da7.png)
_This will have the same options as the WebUI, but as I mentioned, there is a small bug I'll work out today._
